src/electron/find_tap_device_name.bat

```diff
@@ -0,0 +1,80 @@
+:: Copyright 2020 The Outline Authors
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http://www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+
+:: Queries the registry in order to retrieve the most recently installed TAP network adapter's name.
+:: Accepts a single argument, the name of an environment variable to store the adapter's name.
+:: Exits with a non-zero exit code on failure.
+:: Usage example: find_tap_name.bat TAP_NAME
+
+@echo off
+setlocal enabledelayedexpansion
+
+set NET_ADAPTERS_KEY=HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Class\{4D36E972-E325-11CE-BFC1-08002BE10318}
+set NET_CONFIG_KEY=HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Network\{4D36E972-E325-11CE-BFC1-08002BE10318}
+set NET_ADAPTERS_FILE="%tmp%\netadapters.txt"
+
+:: Find all network adapters that match the "tap0901" component ID and store their registry path in a file.
+:: Note that adapter keys are increasing four-digit integers, padded with leading zeros.
+reg query %NET_ADAPTERS_KEY% /s /f "tap0901" /e /d | findstr "HKEY.*\\[0-9][0-9][0-9][0-9]$" > %NET_ADAPTERS_FILE%
+if %errorlevel% neq 0 (
+  echo Could not find TAP network adapter in the registry. >&2
```

Comment:
Is this message going to be visible to a user whose installation fails for this reason?

Reply:
No, this is only visible to us via Sentry. I'm thinking we can display an error code to the user (in a follow up PR).